tests/src/python/test_qgsvectorlayerprofilegenerator.py

```diff
@@ -948,7 +948,7 @@ def testPolygonGenerationTerrain(self):
             self.round_dict(results.distanceToHeightMap(), 1),
             {
                 1041.8: 55.3,
-                1042.4: 55.2,
+                1042.4: 55.3,
```

Comment:
Can you explain why these have changed? The tesselator used in QgsVectorLayerProfileGenerator is set not to calculate normals...

Reply:
The normals are always calculated and used to define the winding order of the resulting triangles. The tessellator _addNormals_ parameter sets whether the normals will be included in the result data, which in this case are not needed.
Still, I'm not exactly sure why the tests needed adjustment. It seems there were some rounding errors caused by the triangles being calculated in the opposite orientation.